src/bp/lang-server/api.ts

```diff
@@ -124,22 +124,38 @@ export default async function(options: APIOptions) {
       domain: options.service.domain,
       readOnly: options.readOnly,
       languages: options.service
-        .listModels()
+        .listFastTextModels()
         .filter(x => x.loaded)
         .map(x => x.name)
     })
   })
 
   app.post('/vectorize', waitForServiceMw, async (req, res, next) => {
+    try {
+      const input = req.body.input
+      const language = req.body.lang || 'en'
```

Comment:
do we really want to default to 'en'. That might not work if the user doesn't have en models... We might want to default to the first model that we both have BP encodings + lang model